runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -720,4 +779,34 @@ private void removeGameTimer(GameTimer timer)
 	{
 		infoBoxManager.removeIf(t -> t instanceof TimerTimer && ((TimerTimer) t).getTimer() == timer);
 	}
+
+	private void removeTbTimers()
+	{
+		removeGameTimer(FULLTB);
+		removeGameTimer(HALFTB);
+		removeGameTimer(DMM_FULLTB);
+		removeGameTimer(DMM_HALFTB);
+	}
+
+	private static boolean isPvpWorld(final EnumSet<WorldType> worldTypes)
+	{
+		return listContainsAnyFromSet(PVP_WORLD_TYPES, worldTypes);
```

Comment:
return worldTypes.stream().anyMatch(w -> PVP_WORLD_TYPES.contains(w));

Reply:
Huh, didn't know you could stream sets. Will do.